src/telliot_feeds/queries/diva_protocol.py

```diff
@@ -84,6 +89,15 @@ class DIVAProtocol(AbiQuery):
 
     def __post_init__(self) -> None:
         """Validate parameters."""
+        if self.poolId is not None:
```

Comment:
Have you considered converting to HexBytes, ie ```try: HexBytes(self.poolId) except:...``` instead of converting to bytes and slicing

Reply:
Good idea, refactored to use HexBytes.